nservicebus/dependency-injection/child-lifetime.md

```diff
@@ -1,30 +1,27 @@
+Child containers are a snapshot of the main container providing an additional instance lifetime scope.
 
-NOTE: As of [NServiceBus.Spring](https://www.nuget.org/packages/NServiceBus.Spring) Version 6.0.0, child lifetimes are now supported as well. Previous versions don't support child lifetimes so one of the other containers supported by NServiceBus must be used to take advantage of it.
 
+## Child container scope in NServiceBus
 
-## Deterministic disposal
-
-Instance lifetime is usually not tracked by the main instance (Windsor is an exception) and that means that manually disposing is required on any instance that needs deterministic disposal. Child lifetimes solve this issue by automatically disposing all transient objects created within each specific child lifetime.
+NServiceBus creates a child container for each transport message that is received. During message processing, all `InstancePerUnitOfWork` scoped instances that are created are resolved as singletons within the context of the same message. This is helpful to support the sharing of database sessions and other resources with lifetimes specific to the processed message.
 
-This is useful to managing things like the database sessions.
+Objects can be configured in a child container scope using the `InstancePerUnitOfWork` lifetime:
 
-NServiceBus creates a child lifetime for each transport message that is received, remembering that transport messages can contain multiple "user defined messages". This means that all transient instances created during message processing are scoped as singletons within the child lifetime. This supports easily sharing, for example, the NHibernate session between repositories, without messing around with thread static caching.
+snippet: InstancePerUnitOfWorkRegistration
 
-When the message finishes processing, the child lifetime and all transient instances are disposed. So if deterministic disposal is required, implement [IDisposable](https://msdn.microsoft.com/en-us/library/system.idisposable.aspx).
 
-When the message is processed, the session is disposed and all resources such as database connections are released.
+## Deterministic disposal
 
-If configuring components using the NServiceBus configure API, it's possible to configure instance lifecycle to be per unit of work, using this:
+Instance lifetime is usually not tracked by the main instance (Windsor is an exception) and that means that manual disposal is required on any instance that needs deterministic disposal. Child containers solve this issue by automatically disposing all instances created within each specific child lifetime. This is useful to managing things like the database sessions.
```

Comment:
The term instance is used three times in this sentence and I suspect (but do not know) that it is referring to at least two different things.

Reply:
I've simplified that section, have another look 👍